user_docs/en/changes.t2t

```diff
@@ -11,6 +11,7 @@ What's New in NVDA
 
 
 == Changes ==
+- The default value for the setting "use screen layout (when supported)" in browse mode settings category changed to disabled (#11716) and the default behavior for the setting "report object when mouse enters it" in the mouse settings category changed to enabled.

```

Comment:
I think these should be separate entries. Please also add relevant issue/PR numbers in the standard format (at the end of the line) and username credit if you'd like